src/modules/mock.command.js

```diff
@@ -0,0 +1,12 @@
+const command = require('./command');
+
+const mock = async function(response, { input, i18n }) {
+	const {user} = response 
```

Comment:
- new empty line after logic block (init
- semicolon on last line
- tabulation problem, use 2 spaces instead 4

Reply:
eslint will solve this kind of issues

Reply:
yes, its need asap